src/device/rdk/applications/launch_with_content.rs

```diff
@@ -239,7 +255,7 @@ pub fn process(_packet: String) -> Result<String, String> {
     let request = RdkRequest {
         jsonrpc: "2.0".into(),
         id: 3,
-        method: "org.rdk.RDKShell.moveToFront".into(),
+        method: "org.rdk.RDKShell.setFocus".into(),
```

Comment:
I can be wrong, but setFocus seems to make it impossible to return to the UI with the remote control home button

Reply:
I see, but your comment says setFocus. And at the same time you did moveToFront twice. Is this intentional?